src/Factory.php

```diff
@@ -30,11 +66,22 @@ public function createLogger($name, array $config)
         if (!method_exists($this, $methodName)) {
             throw new \DomainException(sprintf('Cannot find a logger type named "%s"', $type));
         }
-        $logger   = $this->$methodName($name, $config);
-        $logLevel = isset($config['level']) ? $config['level'] : LogLevel::DEBUG;
-        if ($logLevel !== LogLevel::DEBUG) {
-            return new LevelFilter($logger, $logLevel);
+        $logger = $this->$methodName($name, $config);
+
+        foreach ($this->decorators as $configKey => $decoratorClassName) {
```

Comment:
Perhaps this could be a separate (private) method; `$logger = $this->applyDecorators($logger, $config)`?